### ISSUE TITLE
"Quick instructions" stays when using write deploy keys

### ISSUE THREAD

**Issue by GeoffreyFrogeye** [go-gitea/gitea#3524]
- Gitea version (or commit ref): 1.4.0-rc1, b29c358
- Git version: 2.16.1-1
- Operating system: Arch Linux latest
- Database:
  - [x] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [x] Yes: https://try.gitea.io/GeoffreyFrogeye/writeDeployKey (try cloning the repository, there is a commit)
  - [ ] No
  - [ ] Not relevant
- Log gist: Not relevant

## Description

When you create a new empty repository (so that you get the "Quick instructions" page) and push only using only write-enabled deploy keys, the "Quick instructions" page doesn't go away, even though the repository is still clonable with all its commits.

## Steps to reproduce

- Create a new repository, don't check the checkbox that initialize the repository with files
- Create a SSH key, add it as a write-enabled deploy key to the repository
- Create or use any repository with at least a commit
- Push this repository using the newly created SSH key (beware, depending on your SSH configuration it is really easy to use your account-bound SSH key which does not trigger the bug. I recommend to use a fresh account (either on the gitea server or a system account))
- Contemplate the "Quick instructions" page again
- Try cloning the repository : you get your commits

**Comment by lunny** [comment]
@GeoffreyFrogeye Is this still a problem? I think it has been resolved.

**Comment by GeoffreyFrogeye** [comment]
This is still a problem, I just tested it here: <https://git.frogeye.fr/geoffrey/gitea-3524-test>.

**Comment by zeripath** [comment]
The fix wasn't in 1.7.1 - it's on master and current 1.7 branch.

**Comment by GeoffreyFrogeye** [comment]
Let me try again on <https://try.gitea.io> then.

**Comment by GeoffreyFrogeye** [comment]
It is indeed fixed (see : <https://try.gitea.io/GeoffreyFrogeye/gitea-3524-test>). Thanks!
